network-api/networkapi/buyersguide/factory.py

```diff
@@ -44,7 +44,7 @@ def get_lowest_content_category():
     # TODO: REMOVE: LEGACY FUNCTION
     return sorted(
         [
-            (cat.published_product_count, cat)
+            (cat.published_product_page_count, cat)
```

Comment:
This can stay the old code, since `get_lowest_content_page_category` exists already
```suggestion
            (cat.published_product_count, cat)
```